Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -113,12 +115,13 @@ HRESULT PreparePathForEditing() {
         return S_OK;
     }
 
-    void ClosePath() {
+    HRESULT ClosePath() {
         if (_impl.geometrySink) {
             EndFigure(D2D1_FIGURE_END_OPEN);
```

Comment:
virtual void EndFigure(
   D2D1_FIGURE_END figureEnd
) = 0;

It cannot. Calling it multiple times can result in a bad state though which makes this API slightly weird since it doesn't report that itself.

---
In reply to: [86611357](https://github.com/Microsoft/WinObjC/pull/1307#discussion_r86611357) [](ancestors = 86611357)